ivy/functional/frontends/paddle/tensor/manipulation.py

```diff
@@ -96,3 +96,12 @@ def cast(x, dtype):
 @to_ivy_arrays_and_back
 def broadcast_to(x, shape, name=None):
     return ivy.broadcast_to(x, shape)
+
+
+@with_supported_dtypes(
+    {"2.5.0 and below": ("bool", "float32", "float64", "int32", "int64")},
```

Comment:
`bool` is not registered as a dtype for the `_unstack` kernel in the paddle backend, and so if you see the logs you'll see a runtime error. Can you please remove `bool` from the list and run the test again ?